symphony/lib/toolkit/class.widget.php

```diff
@@ -403,7 +403,7 @@ public static function TableHead(array $columns = null)
                     $th->setAttribute('scope', $col[1]);
                 }
 
-                if (is_array($col[2]) && !empty($col[2])) {
+                if (!empty($col[2]) && is_array($col[2])) {
```

Comment:
`if (General::isNotAnEmptyArray($col[2]))` would hide what ? I feel like it's making the check explicit.